base_layer/wallet/src/utxo_scanner_service/utxo_scanner_task.rs

```diff
@@ -697,9 +698,10 @@ where
         client: &mut BaseNodeWalletRpcClient,
     ) -> Result<HeightHash, UtxoScannerError> {
         let birthday = self.resources.db.get_wallet_birthday()?;
-        // Calculate the unix epoch time of two days before the wallet birthday. This is to avoid any weird time zone
-        // issues
-        let epoch_time = u64::from(birthday.saturating_sub(2)) * 60 * 60 * 24;
+        // Calculate the unix epoch time of two weeks (14 days) before the wallet birthday.
+        // This is to avoid any weird time zone issues
```

Comment:
Time zone issues?

Reply:
Actually, the main reason is to avoid any possible reorgs. I updated the comment to convey this more precisely.